### DISCUSSION TITLE
Tailwind starter blog to Next.Js 13?

### DISCUSSION THREAD

**Discussion by goatednews** [timlrx/tailwind-nextjs-starter-blog#571] [Q&A]
Hello Developers!

Just wanted to know if we're planning to move this template to Next.Js 13?

Thanks.

**Comment by timlrx** [comment]
I am planning to update it to Next 13 with the new app directory structure and React Server Components but there are a few blockers. You can check the progress at https://github.com/timlrx/pliny/issues/80

**Reply by goatednews** [reply]
That would be great. Thanks.

**Reply by goatednews** [reply]
@timlrx Do we have an update on this?

**Comment by germanbobadilla** [comment]
That would be a great idea. I tried adding MDX to a new NextJs 13 template and it was very buggy.
